packages/bodiless-components/src/Editable.static.tsx

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
Should this file be here?  I don't see a .bl-edit version

Reply:
bl-edit version is here [packages/bodiless-components/src/Editable.bl-edit.tsx](https://github.com/wodenx/Bodiless-JS/pull/72/files/f836b1a71a6e45b57fe22310c18838bb1e98a734#diff-f6c71716818b6d30906f742f73b1cfc6a27dc204e1c926c7cbf4c85ff8b0a64a)